From 276abdd6ed72c6dbec7b574a0fcebb31a8a2caf6 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Tue, 17 Apr 2012 14:40:48 -0500
Subject: [PATCH 1343/2508] ENGR00180055 Steer configuration to enable
 AUD5_RXD signal

Steer configuration to enable AUD5_RXD signal

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index c639d32..302f769 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -473,7 +473,7 @@ static int max7310_1_setup(struct i2c_client *client,
 	/* 7 GPS_RST_B */
 
 	int max7310_gpio_value[] = {
-		0, 1, 1, 1, 0, 0, 0, 0,
+		0, 1, 1, 1, 0, 0, 1, 0,
 	};
 
 	int n;
-- 
1.7.7.4

